qiskit_nature/operators/second_quantization/vibrational_op.py

```diff
@@ -150,7 +150,7 @@ def __init__(
             # Sparse label
             dense_labels = self._convert_to_dense_labels(data, num_modals)
 
-            ops: List[VibrationalOp] = []
+            ops: List["VibrationalOp"] = []
```

Comment:
Huh.. I'm surprised to see that this supposedly worked before.. There is also the option to do
```python
from __future__ import annotations
```
What do you think about that?

Reply:
I think the double quotes makes sense as it is what we already use everywhere in typehints with the class being declared inside itself. This fix works with both the old pylint and the new one. It is just that the old didn't report it.

Reply:
For the next release after Terra 0.19 we will drop support for python 3.6. The we can use `from __future__ import annotations` and remove the strings from  the type annotations

Reply:
Right that is only available in python 3.7+.. I hadn't thought of that :sweat_smile: